src/Schema/Events/Search.ts

```diff
@@ -87,3 +87,52 @@ export interface SearchedWithNoResults {
   destination_owner_type?: PageOwnerType
   query: string
 }
+
+/**
+ * A user queries the Artsy Price Database
+ *
+ * This schema describes events sent to Segment from [[searchedPriceDatabase]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "searchedPriceDatabase",
+ *    context_module: "priceDatabaseLanding",
+ *    context_owner_type: "priceDatabase",
+ *    destination_owner_type: "artistAuctionResults",
+ *    destination_owner_id: "4e9746234e77d300010010c6",
+ *    destination_owner_slug: "franz-kline",
+ *    query: "franz kline"
+ *
+ *  }
+ * ```
+ */
+export interface SearchedPriceDatabase {
+  action: ActionType.searchedPriceDatabase
+  context_module: ContextModule.priceDatabaseLanding
+  context_owner_type: OwnerType.priceDatabase
+  destination_owner_type: OwnerType.artistAuctionResults
+  destination_owner_id: string
+  destination_owner_slug: string
+  query: string
```

Comment:
Something to determine: do we pass only the string from the search box or do we include search params? The search params will be on the filtering events (`current`, `changed` props) but I believe we also intend to land users on the artist-insights tab with filters applied, based on their search from the price-database landing page.

Reply:
Looks fine to me to separate filter and search events and properties.
The advantage of having filter and search in the same event is that it will be easier to find the whole query (artist + filters) the user did. For that you could instrument the search button (clicked search button event) and pass both the query and the filters are properties.
But having just filter and search as separate events might more natural and clearer for now!

Q: Does the user has to select an artist similar to our autosuggest experience in the search bar? That might be a simpler event to track and will give more structured data (such as artist id) than the queries which are just text.

Reply:
@louislecluse The autosuggest input works exactly as the "Search by artist name" input on https://www.artsy.net/consign

I think we can either include the query string (where we pass the filters) or the filters (currently applied filters when the user clicks "search") in this event.

Reply:
the fact that this feature works like 'search by artist name' on /consign makes me want to re-do these events to match, especially if we can pass `query` and `filters` when a search is made.

Reply:
> Does the user has to select an artist similar to our autosuggest experience in the search bar? That might be a simpler event to track and will give more structured data (such as artist id) than the queries which are just text.

@louislecluse i think i realize what's bothering me about possibly using 'selectedItemFromSearch' with a context module for priceDatabase, and it's that within `session_tracks` we roll up all 'selectedItemFromSearch' tracks with no regard for context into one measure per session. if we used that event here, we'd be mixing contexts (which are pretty different user flows, imo), and what we're currently meaning to aggregate is the selection of artists from search in the nav bar. 

i think all things considered, i'll do this:
- continue with a new event (searchedPriceDatabase) with properties for the query _and_ the filters passed
- defer to CX about implementing tracks on the filters themselves, since we'll be getting selected filters in the aforementioned track
- defer to CX about implementing focusedOnPriceDatabaseSearchInput, since again what we really need is searchedPriceDatabase, the others are secondarily important I think